gen/com/goide/psi/GoImportSpec.java

```diff
@@ -51,4 +51,6 @@
 
   boolean isCImport();
 
+  boolean isSDKImport();
```

Comment:
It shouldn't be used, but it check import too. It means that you duplicate code by inplementing this method, this is unacceptable